content/patterns/disclosure/examples/css/disclosure-card.css

```diff
@@ -0,0 +1,267 @@
+.disclosure-cards {
+  interpolate-size: allow-keywords;
```

Comment:
I'm not sure how the APG feels about experimental CSS features and no mention of it in the Code Guide. There could be others throughout which makes this moot, but just to point out this gives a slightly different experience visually on Chrome vs Firefox or Safari. It may not matter if the "Read this first" disclaimer at the top generally covers things like this as well, for visitors.

Either way, no negative impact here and I like it in the cases where it is supported :)